backend/src/main/scala/sbt/internal/inc/bloop/internal/BloopNameHashing.scala

```diff
@@ -84,7 +84,10 @@ private final class BloopNameHashing(
           initialChangedSources,
           allSources.map(v => v: VirtualFileRef),
           previous
-        ).collect { case f: VirtualFile => f }
+        ).map {
+          case f: VirtualFile => f
+          case ref: VirtualFileRef => PlainVirtualFileConverter.converter.toVirtualFile(ref)
+        }
```

Comment:
@tgodzik Is `PlainVirtualFileConverter.converter` usage fine here?
I've seen that in some methods `converter` is passed as parameter.

Reply:
I think so, we don't use any other converters currently, so this should be fine.

Reply:
what is the actual type returned? VirtualFileRef is only an interface, no?